### ISSUE TITLE
Pagination: A11y Issues

### ISSUE THREAD

**Issue by daenub** [statistikZH/leu#140]
Siehe A11y Test Output:

```
Error: Accessibility Violations
      ---
      Rule: button-name
      Impact: critical
      Buttons must have discernible text (https://dequeuniversity.com/rules/axe/4.7/button-name?application=axeAPI)

      Issue target: leu-pagination,leu-button[icon="angleLeft"],button
      Context: <button class=" icon secondary normal " disabled="" type="button">
      Fix any of the following:
        Element does not have inner text that is visible to screen readers
        aria-label attribute does not exist or is empty
        aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
        Element has no title attribute
        Element's default semantics were not overridden with role="none" or role="presentation"

      Issue target: leu-pagination,leu-button[icon="angleRight"],button
      Context: <button class=" icon secondary normal " type="button">
      Fix any of the following:
        Element does not have inner text that is visible to screen readers
        aria-label attribute does not exist or is empty
        aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
        Element has no title attribute
        Element's default semantics were not overridden with role="none" or role="presentation"
      ---
      Rule: label
      Impact: critical
      Form elements must have labels (https://dequeuniversity.com/rules/axe/4.7/label?application=axeAPI)

      Issue target: leu-pagination,input
      Context: <input class="input" type="number" min="1" max="14">
      Fix any of the following:
        Form element does not have an implicit (wrapped) <label>
        Form element does not have an explicit <label>
        aria-label attribute does not exist or is empty
        aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
        Element has no title attribute
        Element has no placeholder attribute
        Element's default semantics were not overridden with role="none" or role="presentation"
      ---
        Element does not have inner text that is visible to screen readers
        aria-label attribute does not exist or is empty
        aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
        Element has no title attribute
        Element's default semantics were not overridden with role="none" or role="presentation"
        Issue target: leu-pagination,leu-button[icon="angleRight"],button
        Context: <button class=" icon secondary normal " type="button">
        Fix any of the following:
        Element does not have inner text that is visible to screen readers
        aria-label attribute does not exist or is empty
        aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
        Element has no title attribute
        Element's default semantics were not overridden with role="none" or role="presentation"
        ---
        Rule: label
        Impact: critical
        Form elements must have labels (https://dequeuniversity.com/rules/axe/4.7/label?application=axeAPI)
        Issue target: leu-pagination,input
        Context: <input class="input" type="number" min="1" max="14">
        Fix any of the following:
        Form element does not have an implicit (wrapped) <label>
        Form element does not have an explicit <label>
        aria-label attribute does not exist or is empty
        aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
        Element has no title attribute
        Element has no placeholder attribute
        Element's default semantics were not overridden with role="none" or role="presentation"
```
